src/sentry/grouping/utils.py

```diff
@@ -42,13 +42,31 @@ def get_rule_bool(value):
 def get_fingerprint_value(var, data):
     if var == "transaction":
         return data.get("transaction") or "<no-transaction>"
+    elif var == "message":
+        message = (
+            get_path(data, "logentry", "formatted")
+            or get_path(data, "logentry", "message")
+            or get_path(data, "exception", "values", -1, "value")
+        )
+        return message or "<no-message>"
```

Comment:
I believe this should prioritize `logentry.message` and use the cleaned up message where all replacements of dynamic values have run.

Reply:
The main use of `message` here is to be displayed in the title, not to be used for grouping. This code is kept aligned with how it is used for the title generation.